sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -128,15 +135,18 @@ private[execution] object HashedRelation {
 private[joins] class UnsafeHashedRelation(
     private var numKeys: Int,
     private var numFields: Int,
-    private var binaryMap: BytesToBytesMap)
+    private var binaryMap: BytesToBytesMap,
+    private val isLookupAware: Boolean = false)
   extends HashedRelation with Externalizable with KryoSerializable {
 
-  private[joins] def this() = this(0, 0, null)  // Needed for serialization
+  private[joins] def this() = this(0, 0, null, false)  // Needed for serialization
 
-  override def keyIsUnique: Boolean = binaryMap.numKeys() == binaryMap.numValues()
+  override def keyIsUnique: Boolean = {
```

Comment:
nit: unnecessary change

Reply:
@cloud-fan - sorry, was doing some debugging and forgot to change back. updated.